### PR TITLE
Feature/explore sticky dropdowns

### PR DESCRIPTION
**Pull request by Bluesmile82**
Add a new sticky bar for indicator selection on NDC and LTS explore. 
Don't include the info Button

sticky

### CODE REVIEW

**Comment by Bluesmile82** [comment]
@fannycc do you mean overlapping the search input? The user won't be able to search then. I thought it was part of the table

**Comment by fannycc** [comment]
yes, it is and we should show it (the search input). I understand then that you are making a fixed nav (white) and not only the 2 inputs. So the search input will cover by white fixed space

File: app/javascript/app/components/ndcs/ndcs-explore-map/ndcs-explore-map-component.jsx [hunk 1]
```diff
@@ -76,70 +78,69 @@ const renderLegend = legendData => (
...
-  renderDonutChart = emissionsCardData => (
-    <div
-      className={styles.donutContainer}
-      ref={r => {
-        this.setState({ pieChartRef: r });
-      }}
-    >
+function NDCSExploreMap(props) {
+  const tooltipParentRef = useRef(null);
+  const pieChartRef = useRef(null);
```
Comment (kfilip94) [inline app/javascript/app/components/ndcs/ndcs-explore-map/ndcs-explore-map-component.jsx]: Thanks for using hooks here! :+1:

File: db/secondbase/structure.sql [hunk 2]
```diff
@@ -5,23 +5,10 @@ SET client_encoding = 'UTF8';
...
-CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
-
-
---
--- Name: EXTENSION plpgsql; Type: COMMENT; Schema: -; Owner: -
---
-
-COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';
-
-
```
Comment (kfilip94) [inline db/secondbase/structure.sql]: Are these changes intended?

File: db/structure.sql [hunk 3]
```diff
@@ -5,23 +5,10 @@ SET client_encoding = 'UTF8';
...
-CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
-
-
---
--- Name: EXTENSION plpgsql; Type: COMMENT; Schema: -; Owner: -
---
-
-COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';
-
-
```
Comment (kfilip94) [inline db/structure.sql]: Are these changes intended? [2]

File: db/secondbase/structure.sql (hunk 2 above)
Comment (Bluesmile82) [reply db/secondbase/structure.sql]: 🤔 Maybe I should remove those

Comment (Bluesmile82) [reply db/secondbase/structure.sql]: @simaob are this alright?

Comment (simaob) [reply db/secondbase/structure.sql]: hmmm these look weird, but maybe your version of Postgresql caused them. Might be worth not including them.
